bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.html

```diff
@@ -40,10 +40,10 @@
   <div *ngIf="isInvalidated">
     <p class="m-0 error">This evaluation result has been invalidated.</p>
   </div>
-  <div class="mt-1 mb-1" *ngIf="_selectedEvaluationData" (click)="$event.stopPropagation();">
+  <div class="mt-1 mb-1" *ngIf="_selectedEvaluationData" (click)="$event.stopPropagation();" uitestid="keptn-evaluation-details-selected">
     <div fxLayout="row">
       <div fxFlex></div>
-      <div fxLayout="row" fxLayoutAlign="start end">
+      <div fxLayout="row" fxLayoutAlign="start end" uitestid="keptn-evaluation-details-selected-contextButtons">
         <button dt-button variant="secondary" (click)="showSloDialog()" *ngIf="_selectedEvaluationData.data.evaluation.sloFileContentParsed"><dt-icon name="services"></dt-icon> Show SLO</button>
```

Comment:
Would it make sense to add ids for the buttons like in line 7 and 8?

Reply:
I asked, and the answer was "it depends", but usually it's not needed.